app/Models/UserObyx.php

```diff
@@ -34,11 +34,11 @@ class UserObyx extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
         return $this->belongsTo('App\Models\User', 'user_id', 'id');
     }
 
-    public function obyx(){
+    public function obyx() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->